pkg/util/procfs/procfs.go

```diff
@@ -45,7 +46,15 @@ func containerNameFromProcCgroup(content string) (string, error) {
 // Eg. If the devices cgroup for the container is stored in /sys/fs/cgroup/devices/docker/nginx,
 // return docker/nginx. Assumes that the process is part of exactly one cgroup hierarchy.
 func (pfs *ProcFs) GetFullContainerName(pid int) (string, error) {
-	filePath := path.Join("/proc", strconv.Itoa(pid), "cgroup")
+	var filePath string
+	// if kubelet was started with host's rootfs mounted, assume it's running as a container
+	_, err := os.Stat("/rootfs/proc")
```

Comment:
@pmorie This path could be made configurable too. WDYT? 
+1 for injecting a `containerized` option.